src/components/views/dialogs/ShareDialog.tsx

```diff
@@ -63,10 +63,12 @@ const socials = [
 
 interface BaseProps {
     onFinished(): void;
+    customTitle?: string;
+    subtitle?: string;
 }
 
 interface Props extends BaseProps {
-    target: Room | User | RoomMember;
+    target: Room | User | RoomMember | URL;
```

Comment:
Adding docs for this would be super nice too, although it's fairly obvious once I realised that 'target' was the thing being shared.